pkg/arm/v6/data/master-startup.sh

```diff
@@ -1,5 +1,54 @@
 #!/bin/bash -ex
 
+{{ if gt (len .Config.SecurityPatchPackages) 0 }}
```

Comment:
fwiw you can say `{{ if .Config.SecurityPatchPackages }}` - see https://golang.org/pkg/text/template/#hdr-Actions